editor/editor_vcs_interface.cpp

```diff
@@ -45,6 +45,14 @@ void EditorVCSInterface::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_get_file_diff", "file_path"), &EditorVCSInterface::_get_file_diff);
 	ClassDB::bind_method(D_METHOD("_stage_file", "file_path"), &EditorVCSInterface::_stage_file);
 	ClassDB::bind_method(D_METHOD("_unstage_file", "file_path"), &EditorVCSInterface::_unstage_file);
+	ClassDB::bind_method(D_METHOD("_discard_file", "file_path"), &EditorVCSInterface::_discard_file);
```

Comment:
I thought we wanted to remove this file completely and move everything in the calling place. Do you want to do this later or abandon the idea ?

Reply:
Actually removing default functions would be the effect of making that shift. The file itself will be needed because it contains the definitions of the public functions that Godot calls into